src/annotator/sidebar.js

```diff
@@ -315,6 +346,19 @@ export default class Sidebar extends Guest {
     this.publish('sidebarLayoutChanged', [layoutState]);
   }
 
+  /**
+   * Minimize the sidebar on window resize event
```

Comment:
This comment doesn't seem to quite describe what this callback actually does: it only closes the sidebar if the resulting width is too narrow, right?

Reply:
Yes, you are right. These comments described a previous implementation.